### PR TITLE
Bug/add alt country names 

### PR DESCRIPTION
**Pull request by fivetran-jamie**
## PR Overview
**This PR will address the following Issue/Feature:**
https://github.com/fivetran/dbt_apple_store_source/issues/8

**This PR will result in the following new package version:**

v0.3.0 because the source package has a breaking change (and users of the transform package will also need to re seed the country code data)

**Please detail what change(s) this PR introduces and any additional information that should be known during the review of this PR:**
most of the changes happened in the source PR, but here:
- i also added a Step in the readme about seeding the country code file, since we don't really document as a necessary step anywhere
- we added newly added countries to the one of the seed files so that we can test this out appropriately (ie those countries join properly in the territory report and have non-null data).

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [x] dbt compile
- [x] dbt run –full-refresh
- [x] dbt run
- [x] dbt test
- [ ] dbt run –vars (if applicable)

Before marking this PR as "ready for review" the following have been applied:
- [x] The appropriate issue has been linked and tagged
- [x] You are assigned to the corresponding issue and this PR
- [ ] BuildKite integration tests are passing

### Detailed Validation
Please acknowledge that the following validation checks have been performed prior to marking this PR as "ready for review":
- [x] You have validated these changes and assure this PR will address the respective Issue/Feature.
- [x] You are reasonably confident these changes will not impact any other components of this package or any dependent packages.
- [x] You have provided details below around the validation steps performed to gain confidence in these changes.

validation for this is largely here in the transform package, where i 
1. pointed to the  working branch of the source package
2. added a couple of new countries to one of the territory seed files (specifically Kosovo and two different versions of spelling for the Ivory Coast) 
3. reseeded and reran everything upstream of the territory report 
4. confirmed that the new countries had non-null data from the country code file (so the joins worked)
image
5. confirmed that different spellings of a country will indeed split out into distinct rows (but can be aggregated by their identical country codes)

### Standard Updates
Please acknowledge that your PR contains the following standard updates:
- Package versioning has been appropriately indexed in the following locations:
    - [x] indexed within dbt_project.yml
    - [x] indexed within integration_tests/dbt_project.yml
- [x] CHANGELOG has individual entries for each respective change in this PR
    
- [x] README updates have been applied (if applicable)
    
- [ ] DECISIONLOG updates have been updated (if applicable)
- [x] Appropriate yml documentation has been added (if applicable)

### dbt Docs
Please acknowledge that after the above were all completed the below were applied to your branch:
- [x] docs were regenerated (unless this PR does not include any code or yml updates)

### If you had to summarize this PR in an emoji, which would it be?
 
☔

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,14 @@
-# dbt_apple_store v0.UPDATE.UPDATE
+# dbt_apple_store v0.3.0
 
- ## Under the Hood:
+## Bug Fixes
+[PR #11](https://github.com/fivetran/dbt_apple_store/pull/11) includes the following changes:
+- This version of the transform package points to a [breaking change in the source package](https://github.com/fivetran/dbt_apple_store_source/blob/main/CHANGELOG.md) in which the [country code](https://github.com/fivetran/dbt_apple_store_source/blob/main/seeds/apple_store_country_codes.csv) mapping table was updated to align with Apple's [format and inclusion list](https://developer.apple.com/help/app-store-connect/reference/app-store-localizations/) of country names.
+  - This is a 🚨**breaking change**🚨 as you will need to re-seed (`dbt seed --full-refresh`) the `apple_store_country_codes` file again.
 
+## Under the Hood:
```
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md:8]: Would you be able to provide the PR reference for the below updates.

File: README.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ Include the following apple_store package version in your `packages.yml` file:
 ```yaml
 packages:
   - package: fivetran/apple_store
-    version: [">=0.2.0", "<0.3.0"] # we recommend using ranges to capture non-breaking changes automatically
+    version: [">=0.3.0", "<0.4.0"] # we recommend using ranges to capture non-breaking changes automatically
```
Comment (fivetran-joemarkiewicz) [inline README.md:51]: Can we also add the same README update you applied to the source package to use the seed file for country codes effectively?
image

File: packages.yml [hunk 3]
```diff
@@ -1,3 +1,6 @@
 packages:
-- package: fivetran/apple_store_source
-  version: [">=0.2.0", "<0.3.0"]
+# - package: fivetran/apple_store_source
+#   version: [">=0.3.0", "<0.4.0"]
+- git: https://github.com/fivetran/dbt_apple_store_source.git
+  revision: bug/add-alt-country-names
+  warn-unpinned: false
```
Comment (fivetran-joemarkiewicz) [inline packages.yml:6]: Reminder to switch before release.

**Review by fivetran-joemarkiewicz — APPROVED** [review]
@fivetran-jamie this looks good to go! I only have a few minor requests to update some documentation similar to how you made updates in the original source package. Thanks for working through these!

File: README.md (hunk 2 above)
Comment (fivetran-jamie) [reply README.md:51]: added

File: CHANGELOG.md (hunk 1 above)
Comment (fivetran-jamie) [reply CHANGELOG.md:8]: added

**Pull request merged**